### ISSUE TITLE
htmlcs: display: hidden & visibility: hidden issues evaluated

### ISSUE THREAD

**Issue by metsis** [a11ywatch/kayle#27]
Hi,

We're are finding that elements that should be clearly hidden both visually and from screen readers (`display:none; visibility: hidden`) are being evaluated as accessibility errors. Shouldn't they be ignored, or is there an option to choose to do so? Or is this a deliberate choice?

```
{"context":"<iframe height=\"0\" width=\"0\" style=\"display: none; visibility: hidden;\" data-lf-form-tracking-inspected-laxoeakglgaaoygd=\"true\" data-lf-yt-playback-inspected-laxoeakglgaaoygd=\"true\" data-lf-vimeo-playback-inspected-laxoeakglgaaoygd=\"true\"></iframe>",
"selector":"body>:nth-child(7)",
"code":"WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1",
"type":"error",
"typeCode":1,
"message":"Iframe element requires a non-empty title attribute that identifies the frame.",
"runner":"htmlcs",
"recurrence":0,
"clip":{"x":0,"y":0,"width":0,"height":0}}
```

**Comment by j-mendez** [comment]
Hi, They should be ignored, depending on the issue. This is a meta issue that needs a title regardless of visibility.
